### ISSUE TITLE
semicolon rule: "never" option doesn't detects semicolon in export default expression

### ISSUE THREAD

**Issue by Guria** [palantir/tslint#1155]
### Bug Report

- __TSLint version__: `3.8.0`
- __TypeScript version__: `1.8.10`
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
export default LoginPage;
```

with `tslint.json`:

```json
{
  "rules": {
    "semicolon": [true, "never"]
  }
}
```

#### Actual behavior

No error

#### Expected behavior

Should output error `unnecessary semicolon`

**Comment by adidahiya** [comment]
Thanks for the bug report @Guria! Should be a relatively easy fix if anyone wants to send a PR.
